temporalio/worker/workflow_instance.py

```diff
@@ -0,0 +1,1314 @@
+                )
+
+            # Schedule it
+            # TODO(cretz): Do we need to do this inside the activity task so it
+            # can be cancelled?
+            self.create_task(backoff_then_reschedule())
+        else:
+            raise RuntimeError("Activity did not have result")
+
+    def _apply_resolve_child_workflow_execution(
+        self,
+        job: temporalio.bridge.proto.workflow_activation.ResolveChildWorkflowExecution,
+    ) -> None:
+        # No matter the result, we know we want to pop
+        handle = self._pending_child_workflows.pop(job.seq, None)
+        if not handle:
+            raise RuntimeError(
+                f"Failed finding child workflow handle for sequence {job.seq}"
+            )
+        if job.result.HasField("completed"):
+            ret: Optional[Any] = None
+            if job.result.completed.HasField("result"):
+                ret_types = [handle._input.ret_type] if handle._input.ret_type else None
+                ret_vals = self._convert_payloads(
+                    [job.result.completed.result],
+                    ret_types,
+                )
+                ret = ret_vals[0]
+            handle._resolve_success(ret)
+        elif job.result.HasField("failed"):
+            handle._resolve_failure(
+                temporalio.exceptions.failure_to_error(
+                    job.result.failed.failure, self._payload_converter
+                )
+            )
+        elif job.result.HasField("cancelled"):
+            handle._resolve_failure(
+                temporalio.exceptions.failure_to_error(
+                    job.result.cancelled.failure, self._payload_converter
+                )
+            )
+        else:
+            raise RuntimeError("Child workflow did not have result")
+
+    def _apply_resolve_child_workflow_execution_start(
+        self,
+        job: temporalio.bridge.proto.workflow_activation.ResolveChildWorkflowExecutionStart,
+    ) -> None:
+        handle = self._pending_child_workflows.get(job.seq)
+        if not handle:
+            raise RuntimeError(
+                f"Failed finding child workflow handle for sequence {job.seq}"
+            )
+        if job.HasField("succeeded"):
+            handle._resolve_start_success(job.succeeded.run_id)
+        elif job.HasField("failed"):
+            self._pending_child_workflows.pop(job.seq)
+            if (
+                job.failed.cause
+                == temporalio.bridge.proto.child_workflow.StartChildWorkflowExecutionFailedCause.START_CHILD_WORKFLOW_EXECUTION_FAILED_CAUSE_WORKFLOW_ALREADY_EXISTS
+            ):
+                handle._resolve_failure(
+                    temporalio.exceptions.WorkflowAlreadyStartedError(
+                        job.failed.workflow_id, job.failed.workflow_type
+                    )
+                )
+            else:
+                handle._resolve_failure(
+                    RuntimeError("Unknown child start fail cause: {job.failed.cause}")
```

Comment:
```suggestion
                    RuntimeError(f"Unknown child start fail cause: {job.failed.cause}")
```

Reply:
Done

Reply:
Is there some lint rule that can identify this?

Reply:
Hrmm, maybe. Currently I only use a style linter and MyPy and a doc linter. I don't have a general purpose linter for general Python suggestions. I will see if I can find a well-accepted one.

Reply:
I checked `pylint` and confirmed it wouldn't catch this. I suppose there would be too many false positives with normal brace use in string literals.

I also just gave `pylint` a general look and it doesn't give us more than MyPy really, it's actually worse (can't recognize stub files). And black solves the formatting. So I think we're as good as we can reasonably get with linting currently.